packages/foundry.streams/src/_components.ts

```diff
@@ -59,6 +59,29 @@ export interface Dataset {
  */
 export type PartitionsCount = number;
 
+/**
+ * Log Safety: DO_NOT_LOG
+ */
+export interface PublishRecordToStreamRequest {
+  record: _Record;
+  viewRid?: ViewRid;
+}
+
+/**
+ * Log Safety: DO_NOT_LOG
+ */
+export interface PublishRecordsToStreamRequest {
+  records: Array<_Record>;
+  viewRid?: ViewRid;
+}
+
+/**
+ * A record to be published to a stream.
+ *
+ * Log Safety: DO_NOT_LOG
+ */
+export type _Record = Record<string, any>;
```

Comment:
So they declared a type called Record, which is why I had to do some weird overrides in our codegen to not conflict with TS record types